content/blog/dbplyr-2-0-0/index.Rmd

```diff
@@ -0,0 +1,204 @@
+---
+output: hugodown::hugo_document
+
+slug: dbplyr-2-0-0
+title: dbplyr 2.0.0
+date: 2020-11-03
+author: Hadley Wickham
+description: >
+    dbplyr 2.0.0 brings dplyr 1.0.0 compatibility, numeric improvements
```

Comment:
the "dbplyr 2.0.0 brings dplyr 1.0.0 compatibility" construct is a little difficult to parse just because the words are so similar that it almost reads as "x 2.0.0 brings x 1.0.0 compatibility"

I don't have a great alternative suggestion, maybe something with bringing dbplyr up to speed with dplyr 1.0? I'll keep thinking on this one.

Reply:
Double checking that you actually mean "numeric improvements" and not numerous. If you _do_ mean numeric, I'd make the addition of new backends its own item in the list.

Reply:
I did mean numerous 😬